rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java

```diff
@@ -656,7 +656,7 @@ private Integer addFederatorPublicKeyMultikey(boolean dryRun, BtcECKey btcKey, E
      * PENDING_FEDERATION_MISMATCHED_HASH if the given hash doesn't match the current pending federation's hash.
      * SUCCESSFUL upon success.
      */
-    private FederationChangeResponseCode commitFederation(boolean dryRun, Keccak256 pendingFederationHash, BridgeEventLogger eventLogger) {
+    FederationChangeResponseCode commitFederation(boolean dryRun, Keccak256 pendingFederationHash, BridgeEventLogger eventLogger) {
```

Comment:
Can't we access this method by calling `voteFederationChange`?

I think we should test this sending the required votes. Make it as realistic as possible. See `VoteFederationChangeTest` class